content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,121 @@
+---
+title: June '21 Community Gems
+date: 2021-06-25
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-05-25/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/june-21-community-gems/779
+tags:
```

Comment:
Please, let's take a look what tags we are usually using for Gems - I think, at least Gems is missing? something else?

Reply:
It didn't look like the other Gems have the "Gems" tag. They all had pretty similar things like, "Community", "CML", "Pipelines". I did update these though.

Reply:
Hmm here:

<img width="523" alt="Screen Shot 2021-06-30 at 8 01 15 PM" src="https://user-images.githubusercontent.com/3659196/124058297-f668df80-d9dd-11eb-9c53-e71efd93fcfc.png">


?

https://dvc.org/blog/may-21-community-gems

Reply:
really minor one, @flippedcoder - should we address this one ?